cost/google/cud_recommendations/CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 # Changelog
 
-## v3.1
+## v2.2
 
 - Fixed missing savings field
 - updated changelog ordering
 
-## v3.0
+## v2.1
```

Comment:
Isn't the 3.x version ok, I thought that's consistent with what we did when refactoring the other policies to follow the field names for spend recommendations.

Reply:
I see, just matching the other Google recommendation policies, 2.x.